example/spec/parallel/testAPI/ruleSpec.js

```diff
@@ -17,20 +17,6 @@ const config = loadConfig();
 
```

Comment:
I thought it might be easiest to throw my tests in with the existing API tests. My other option is to create a completely different spec file for `ScheduledRules`. I'd like to hear what others think.